.github/workflows/e2e_dart.yml

```diff
@@ -0,0 +1,30 @@
+name: e2e-sentry-dart
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  build:
+    name: E2E
+    runs-on: 'ubuntu-latest'
+    defaults:
+      run:
+        working-directory: ./e2e_test
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable, beta, dev]
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+
+    steps:
+        - uses: dart-lang/setup-dart@v1
+          with:
+            sdk: ${{ matrix.sdk }}
+        - uses: actions/checkout@v2
+        - name: Test 
+          run: |
+            dart pub get
+            dart run
```

Comment:
can we run the program via Dart VM and Dart JS? the idea is to caught problems for the 2 environments

Reply:
I would like tod o that in another PR.
For web we have to spin up a browser, a development server etc.